app/routes/client.py

```diff
@@ -183,24 +192,28 @@ def event_archive(index):
 
     :param int index: The page to fetch
     """
-    index = int(index)
     if index <= 0:
         return redirect(url_for('.events'))
 
+    # Get all events that occur on this page or on subsequent pages
     today = date.today()
-    last_sunday = datetime.combine(today - timedelta(days=today.weekday()+7),
-                                   datetime.min.time())
+    events = (Event.objects(published=True, end_date__lt=today)
+                   .order_by('-start_date')
```

Comment:
So this is just specialized Mongo knowledge, right?
It doesn't seem right to require this comment everywhere but it's definitely not obvious. Should there be a general doc/readme on special features or what?

(this is a common internal debate I have with using advanced features of things on ADI projects, i usually just document profusely)